backend/Dockerfile

```diff
@@ -1,12 +1,10 @@
-FROM quay.io/centos/centos:8
+FROM registry.redhat.io/rhel8/python-39
 
-COPY . /backend
+COPY --chown=0:0 backend /backend
```

Comment:
The excerpt from [the documentation](https://docs.docker.com/engine/reference/builder/#copy) seems to say the default ownership for copy commands is `0:0`:
```
All new files and directories are created with a UID and GID of 0, unless the optional --chown flag specifies a given username, groupname, or UID/GID combination to request specific ownership of the copied content. 
```
What is the requirement for the `--chown=0:0`?

Reply:
Good catch. I was not aware of that. Changed it now.